src/test/java/edu/unc/lib/boxc/migration/cdm/SourceFilesCommandIT.java

```diff
@@ -249,10 +249,69 @@ public void statusUnmappedVerboseTest() throws Exception {
         assertOutputMatches(".*Potential Matches: +0.*");
     }
 
+    @Test
+    public void statusUnmappedDoesNotContainGroupObjectsTest() throws Exception {
+        indexGroupExportSamples();
+        addSourceFile("276_182_E.tif");
+        addSourceFile("276_203_E.tif");
+
+        String[] args = new String[] {
+                "-w", project.getProjectPath().toString(),
+                "source_files", "generate",
+                "-b", basePath.toString()};
+        executeExpectSuccess(args);
+
+        String[] args2 = new String[] {
+                "-w", project.getProjectPath().toString(),
+                "group_mapping", "generate",
+                "-n", "groupa"};
+        executeExpectSuccess(args2);
+
+        String[] args3 = new String[] {
+                "-w", project.getProjectPath().toString(),
+                "group_mapping", "sync" };
+        executeExpectSuccess(args3);
+
+        String[] args4 = new String[] {
+                "-w", project.getProjectPath().toString(),
+                "source_files", "status"};
+        executeExpectSuccess(args4);
+
+        assertOutputMatches(".*Unmapped Objects: +3.*");
+    }
+
+    @Test
+    public void statusUnmappedDoesNotContainCompoundObjectsTest() throws Exception {
+        indexCompoundExportSamples();
+        addSourceFile("276_182_E.tif");
```

Comment:
for the sake of having a realistic test, we should use filenames that match the collection being used (the filenames listed here are from gilmer). All of the filenames can be seen here:
https://github.com/UNC-Libraries/cdm-to-boxc-migration-util/blob/main/src/test/java/edu/unc/lib/boxc/migration/cdm/services/SipServiceTest.java#L546-L549
We'll also need to specify the correct field to use when calling `source_files generate` below, since it needs to use the `filena` field, like in the example i linked. If we include all these, then the number of unmapped objects should end up as 0, which should clearly illustrate that the compound objects aren't being reported.